package.json

```diff
@@ -47,6 +47,7 @@
     "eslint": "5.9.0",
     "eslint-config-airbnb": "17.1.0",
     "eslint-config-prettier": "3.3.0",
+    "eslint-import-resolver-webpack": "^0.10.1",
```

Comment:
please install only with -E option (exact version)

Reply:
or you can use `npm i <package-name> --save-exact` , it's equivalent to -E